src/components/spreadsheet/utils/config-tables.js

```diff
@@ -91,6 +96,29 @@ export const DEFAULT_SORT_ORDER = 'asc';
 
```

Comment:
You can remove DEFAULT_SORT_ORDER (which is located just above this line but I can't comment it) now that it is no longer in use